src/next-transpile-modules.js

```diff
@@ -120,82 +163,37 @@ const withTmInitializer = (transpileModules = [], options = {}) => {
           );
 
           if (nextCssLoader) {
-            nextCssLoader.issuer.or = nextCssLoader.issuer.and ? nextCssLoader.issuer.and.concat(includes) : includes;
-            nextCssLoader.issuer.not = excludes;
+            nextCssLoader.issuer.or = nextCssLoader.issuer.and ? nextCssLoader.issuer.and.concat(match) : match;
+            nextCssLoader.issuer.not = [unmatch];
             delete nextCssLoader.issuer.and;
+          } else {
+            console.warn('next-transpile-modules: could not find default CSS rule, CSS imports may not work');
           }
 
           if (nextSassLoader) {
-            nextSassLoader.issuer.or = nextSassLoader.issuer.and
-              ? nextSassLoader.issuer.and.concat(includes)
-              : includes;
-            nextSassLoader.issuer.not = excludes;
+            nextSassLoader.issuer.or = nextSassLoader.issuer.and ? nextSassLoader.issuer.and.concat(match) : match;
+            nextSassLoader.issuer.not = [unmatch];
```

Comment:
Hello, I used the @ScriptedAlchemy module (@module-federation/next-transpile-modules@5.0.0-beta.6) which contains this code and this line is breaking import of CSS modules in src/ components.
I can create a repo with my issue if you need.

Reply:
@noreiller Do you mean @module-federation or my line is breaking your build? or both?

Reply:
Sorry for misunderstanding, this line (`nextSassLoader.issuer.not = [unmatch];` ) avoids the imports of CSS modules in the files which comply to `unmatch`. If I comment it, I can use them. 
I import CSS modules in my `src/` components and in some `node_modules/` components.

Reply:
Ah I see, you mean normal CSS files from your application (that are not in node_modules)

Reply:
Yes.

Reply:
Thank you, it should not be fixed 🙌

Reply:
Thank you !